### DISCUSSION TITLE
Hi I love this extension - but how do I get it to keep the windows buttons and icons as well as this?

### DISCUSSION THREAD

**Discussion by arunk** [michaelbrusegard/tabline.wez#3] [Q&A] [answered]
I love this extension for wezterm. But I realize it completely replaces all the windows controls and tabs. Is there a way I can keep some or all of those windows buttons - like the + button that adds a new tab or the close button for the window?

**Comment by michaelbrusegard** [comment, marked as answer]
Yes! The plugin enables some recommended settings when you use the `apply_to_config(config)` method. You do not have to call this method if you do not want to. These are the options that the method sets for you:

```lua
function M.apply_to_config(config)
  config.use_fancy_tab_bar = false
  config.show_new_tab_button_in_tab_bar = false
  config.tab_max_width = 32
  config.window_decorations = 'RESIZE'
  config.window_padding = {
    left = 0,
    right = 0,
    top = 0,
    bottom = 0,
  }
  config.colors = {
    tab_bar = {
      background = require('tabline.config').colors.normal_mode.c.bg,
    },
  }
  config.status_update_interval = 500
end
```

The only option you really need to set is: `config.use_fancy_tab_bar = false`. To keep the + button avoid setting `config.show_new_tab_button_in_tab_bar = false` and to keep the windows controls and buttons avoid setting: `config.window_decorations = 'RESIZE'`. I hope this answers your question!
